closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -82,6 +82,9 @@ int OSSL102_RSA_set0_crt_params(RSA *, BIGNUM *, BIGNUM *, BIGNUM *);
 #define EVP_CTRL_AEAD_SET_TAG EVP_CTRL_GCM_SET_TAG
 #endif
 
+/* Check whether loaded library is in FIPS mode. */
```

Comment:
"Check" is not appropriate here (a variable is passive). Suggest this flag be `static`:
```c
/* Whether loaded library is in FIPS mode. */
static jboolean OSSL_IS_FIPS;
```
(Most other globals in this file should be `static`, but that might best be corrected separately.)

Reply:
Changed comment and added `static`.